apps/web/pages/[user].tsx

```diff
@@ -312,6 +316,18 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
     avatar: `/${user.username}/avatar.png`,
   }));
 
+  if (!isOrgContext) {
+    const redirect = await getTemporaryOrgRedirect({
+      slug: usernameList[0],
```

Comment:
Not supporting redirect for dynamic booking link for now as they would most probably be not hardcoded anywhere.